cmake/UnitTests.cmake

```diff
@@ -16,9 +16,10 @@ list(APPEND GPU_POCL_FAILED_TESTS " ")  # TODO
 # It fails with "error: cannot find ROCm device library;
 #  provide its path via '--rocm-path' or '--rocm-device-lib-path', or pass
 #  '-nogpulib' to build without ROCm device library"
-list(APPEND FAILING_FOR_ALL "Unit_hipMalloc_LoopRegressionAllocFreeCycles") # 
-list(APPEND FAILING_FOR_ALL "Unit_hipGraphAddHostNode_ClonedGraphwithHostNode") #
-list(APPEND FAILING_FOR_ALL "Unit_hipGraphDestroyNode_DestroyDependencyNode") #
+list(APPEND FAILING_FOR_ALL "abort") # 
```

Comment:
Don't these work with OpenCL in your problematic platform? So, excluding in the L0 also should work? Can you also add some comments here so we remember what you've found of the failure?